src/api/pdfrenderer.cpp

```diff
@@ -623,24 +625,21 @@ bool TessPDFRenderer::BeginDocumentHandler() {
 
   stream.str("");
   stream << datadir_.c_str() << "/pdf.ttf";
```

Comment:
Wouldn't it be cleaner to remove the code for reading `pdf.ttf` and remove that file from installations, too?

Reply:
I leave it there if somebody would like to adapt and test pdf.ttf

Reply:
Would it be sufficient to enable that feature only for `DEBUG` code? For most users (and that includes all Linux distributions) installing `pdf.ttf` is not necessary, and I'd prefer to remove it from the normal installation.

Reply:
I think using it in DEBUG code only it good idea.